instance/templates/instance/ansible/vars.yml

```diff
@@ -166,3 +202,42 @@ POSTFIX_QUEUE_HEADER_CHECKS: '/^From:(.*)$/   PREPEND Reply-To:$1'
 POSTFIX_QUEUE_SENDER_CANONICAL_MAPS: '{{ smtp_relay_settings.source_address }}  {{ smtp_relay_settings.rewritten_address }}'
 {% endif %}
 {% endwith %}
+
+# Ecommerce
+SANDBOX_ENABLE_ECOMMERCE: false  # set to true to enable ecommerce
+ECOMMERCE_ECOMMERCE_URL_ROOT: '{{ instance.protocol }}://{{ instance.domain }}:18130'
+ECOMMERCE_SUPPORT_URL: '{{ instance.protocol }}://{{ instance.domain }}'
+ECOMMERCE_LMS_URL_ROOT: '{{ instance.domain }}'
+ECOMMERCE_PLATFORM_NAME: "{{ EDXAPP_PLATFORM_NAME }}"
+# Set your own ECOMMERCE_PAYMENT_PROCESSOR_CONFIG, or update the variables for the appropriate processor:
+# * CyberSource
+ECOMMERCE_CYBERSOURCE_PROFILE_ID: 'SET-ME-PLEASE'
+ECOMMERCE_CYBERSOURCE_MERCHANT_ID: 'SET-ME-PLEASE'
+ECOMMERCE_CYBERSOURCE_ACCESS_KEY: 'SET-ME-PLEASE'
+ECOMMERCE_CYBERSOURCE_SECRET_KEY: 'SET-ME-PLEASE'
+ECOMMERCE_CYBERSOURCE_TRANSACTION_KEY: 'SET-ME-PLEASE'
+ECOMMERCE_CYBERSOURCE_PAYMENT_PAGE_URL: 'https://set-me-please'
+ECOMMERCE_CYBERSOURCE_RECEIPT_PAGE_URL: '{{ ECOMMERCE_LMS_URL_ROOT }}/commerce/checkout/receipt/'
+ECOMMERCE_CYBERSOURCE_CANCEL_PAGE_URL: '{{ ECOMMERCE_LMS_URL_ROOT }}/commerce/checkout/cancel/'
+ECOMMERCE_CYBERSOURCE_SOAP_API_URL: 'https://set-me-please'
+# * PayPal
+ECOMMERCE_PAYPAL_MODE: 'SET-ME-PLEASE'
+ECOMMERCE_PAYPAL_CLIENT_ID: 'SET-ME-PLEASE'
+ECOMMERCE_PAYPAL_CLIENT_SECRET: 'SET-ME-PLEASE'
+ECOMMERCE_PAYPAL_RECEIPT_URL: '{{ ECOMMERCE_LMS_URL_ROOT }}/commerce/checkout/receipt/'
+ECOMMERCE_PAYPAL_CANCEL_URL: '{{ ECOMMERCE_LMS_URL_ROOT }}/commerce/checkout/cancel/'
+ECOMMERCE_PAYPAL_ERROR_URL: '{{ ECOMMERCE_LMS_URL_ROOT }}/commerce/checkout/error/'
+
+# Comprehensive theming
+# EDXAPP_ENABLE_COMPREHENSIVE_THEMING: true
+# EDXAPP_COMPREHENSIVE_THEME_DIRS:
+#   - '/edx/app/edxapp/themes'
+# EDXAPP_DEFAULT_SITE_THEME: 'mythemename'
+# edxapp_theme_name: '{{ EDXAPP_DEFAULT_SITE_THEME }}'
+# edxapp_theme_source_repo: 'https://github.com/open-craft/comprehensive-theme.git'
```

Comment:
@mtyaka Should the repo name in the example URL match the value of the `EDXAPP_DEFAULT_SITE_THEME` setting above (`https://github.com/open-craft/my-theme-name.git`)?